core-service/src/test/java/com/crmfoodestablishment/coreservice/service/MenuServiceTest.java

```diff
@@ -17,109 +16,109 @@
 import static org.mockito.BDDMockito.given;
 import static org.mockito.Mockito.*;
 
-@ExtendWith(MockitoExtension.class)
```

Comment:
If these test are invalid - change them so they would work. Or simply don't commit it :)